include/spdlog/fmt/bundled/format.h

```diff
@@ -3260,7 +3260,10 @@ void BasicWriter<Char>::write_int(T value, Spec spec)
     case 'n':
     {
         unsigned num_digits = internal::count_digits(abs_value);
-        fmt::StringRef sep = internal::thousands_sep(std::localeconv());
+        fmt::StringRef sep = "";
+#ifndef ANDROID
```

Comment:
Is this fix is already in fmt lib ? If yes I will take it from there directly, so please remove it from this file